docs/root/start/simple_example.rst

```diff
@@ -0,0 +1,259 @@
+      listeners:
+      - name: listener_0
+        address:
+          socket_address: { address: 0.0.0.0, port_value: 10000 }
+        filter_chains:
+        - filters:
+          - name: envoy.http_connection_manager
+            config:
+              stat_prefix: ingress_http
+              route_config:
+                name: local_route
+                virtual_hosts:
+                - name: local_service
+                  domains: ["*"]
+                  routes:
+                  - match: { prefix: "/" }
+                    route: { host_rewrite: www.google.com, cluster: service_google }
+              http_filters:
+              - name: envoy.router
+      clusters:
+      - name: service_google
+        connect_timeout: 0.25s
+        type: LOGICAL_DNS
+        # Comment out the following line to test on v6 networks
+        dns_lookup_family: V4_ONLY
+        lb_policy: ROUND_ROBIN
+        hosts: [{ socket_address: { address: google.com, port_value: 443 }}]
+        tls_context: { sni: www.google.com }
+
+
+Let's go through each section in detail.
+
+.. code-block:: yaml
+
+  admin:
+    access_log_path: /tmp/admin_access.log
+    address:
+      socket_address: { address: 0.0.0.0, port_value: 9901 }
+
+..
+
+  The :ref:`admin message <envoy_api_msg_config.bootstrap.v2.Admin>` is required to configure
+  the administration server.
+
+  The ``address`` key specifies the
+  listening :ref:`address <envoy_api_file_envoy/api/v2/core/address.proto>`
+  which in this case is simply `0.0.0.0:9901`.
+
+.. code-block:: yaml
+
+    static_resources:
+
+..
+
+  The :ref:`static_resources <envoy_api_field_config.bootstrap.v2.Bootstrap.static_resources>` contains
+  everything that is configured statically when Envoy starts,
+  as opposed to the means of configuring resources dynamically when Envoy is running.
+  The :ref:`v2 API Overview <config_overview_v2>` describes this.
+
+.. code-block:: yaml
+
+      listeners:
+      - name: listener_0
+        address:
+          socket_address: { address: 0.0.0.0, port_value: 10000 }
+        filter_chains:
+
+..
+
+  The specification of the :ref:`listeners <envoy_api_file_envoy/api/v2/lds.proto>`. One or more
+  listeners can be specified as members of ``listeners``.
+
+  The ``name`` key uniquely identifies the listener.
+
+  The ``address`` key specifies specifies the
+  listening :ref:`address <envoy_api_file_envoy/api/v2/core/address.proto>`
+  which in this case is simply `0.0.0.0:10000`.
+
+  The ``filter_chains`` key is required and specifies the list of filter chains to
+  consider for this listener. The
+  :ref:`FilterChain <envoy_api_msg_listener.FilterChain>` with the most specific
+  :ref:`FilterChainMatch <envoy_api_msg_listener.FilterChainMatch>` criteria is used on a
+  connection.
+
+.. code-block:: yaml
+
+        - filters:
+          - name: envoy.http_connection_manager
+            config:
+
+..
+
+  The filter chain consists of a list of ``filters``.
+
```

Comment:
Similarly, we could link to https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/http_filters and https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/network_filters for more context.